src/jdk.compiler/share/classes/com/sun/tools/javac/parser/DocCommentParser.java

```diff
@@ -249,21 +406,72 @@ protected List<DCTree> blockContent(Phase phase) {
                         textStart = bp;
                         break;
                     }
-                    // fallthrough
+                    defaultContentCharacter();
+                }
 
-                default:
-                    newline = false;
-                    if (textStart == -1)
-                        textStart = bp;
-                    lastNonWhite = bp;
-                    nextChar();
+                case '\\' -> {
+                    switch (textKind) {
+                        case MARKDOWN -> {
+                            defaultContentCharacter();
+                            nextChar();
+                            defaultContentCharacter();
+                        }
+                        case TEXT -> {
+                            defaultContentCharacter();
+                        }
+                    }
```

Comment:
Is it to pass `\` through to Markdown, to allow it to deal with Markdown escapes?

Reply:
You might also need to use `` \` `` when there are two literal backticks in a sentence.
```
After the first backtick (\`), another backtick (\`) can be used to delimit a code span.
```